explainer.md

```diff
@@ -424,9 +424,9 @@ If the input source has only 3DOF, the grip matrix may represent only a translat
 
 An input source will also provide its preferred target ray on its pose, which is defined as a ray originating at `[0, 0, 0]` and extending down the negative Z axis, transformed by the `targetRayMatrix` attribute of an `XRInputPose` object. `targetRayMatrix` will never be `null`. The value will differ based on the type of input source that produces it, which is represented by the `targetRayMode` attribute:
 
-  * `'gazing'` indicates the target ray will originate at the user's head and follow the direction they are looking (this is commonly referred to as a "gaze input" device). While it may be possible for these devices to be tracked (and have a grip matrix), the head gaze is used for targeting. Example devices: 0DOF clicker, regular gamepad, voice command, tracked hands.
-  * `'pointing'` indicates that the target ray originates from a handheld device and represents that the user is using that device for pointing. The exact orientation of the ray relative to the device should follow platform-specific guidelines if there are any. In the absence of platform-specific guidance, the target ray should most likely point in the same direction as the user's index finger if it was outstretched while holding the controller.
-  * `'tapping'` indicates that the input source was an interaction with the 2D canvas of a non-immersive session, such as a mouse click or touch event. See [Magic Window Input](#magic_window_input) for more details.
+  * `'gaze'` indicates the target ray will originate at the user's head and follow the direction they are looking (this is commonly referred to as a "gaze input" device). While it may be possible for these devices to be tracked (and have a grip matrix), the head gaze is used for targeting. Example devices: 0DOF clicker, regular gamepad, voice command, tracked hands.
+  * `'handheld'` indicates that the target ray originates from a handheld device and represents that the user is using that device for pointing. The exact orientation of the ray relative to the device should follow platform-specific guidelines if there are any. In the absence of platform-specific guidance, the target ray should most likely point in the same direction as the user's index finger if it was outstretched while holding the controller.
```

Comment:
`handheld` captures my original intent well, I like it. Wondering if it is worth considering simply `held`?

Reply:
I'm a bit concerned that `held` or `handheld` will make the addition of actual hands a bit weird.  I don't have an alternative to suggest though.
Also, what about `screen` instead of `canvas`?

Reply:
I have no objection to `screen`. I think I recall someone saying at some point that it may not be strictly accurate because you may be interacting with a virtual surface rather than a physical screen, but I doubt that distinction is going to cause much confusion among developers.

Not sure how I feel about `held` other than the fact that it has the same downsides as `handheld` without the word `hand`, which seems like a step in the wrong direction. (All for shorter enums, though!) Again, maybe `hand` is the best option here? Which, if we combined that with the `screen` change would mean we've basically just reverted the enums to their prior state but changed `head` to `gaze`. :smile: That may be OK, though, since the context surrounding them has changed a bit for the better.